src/component/tooltip/TooltipHTMLContent.ts

```diff
@@ -336,7 +337,7 @@ class TooltipHTMLContent {
             self._inContent = false;
 
             if (self._enterable) {
-                if (self._show) {
+                if (self._show && !self._alwaysShowContent) {
```

Comment:
I think it's more rust to check this in `hideLater` rather than `onmouseleave` because when other reasons triggers hiding, this should also be checked.

Reply:
get it